plugins/net_plugin/net_plugin.cpp

```diff
@@ -808,6 +808,7 @@ namespace eosio {
       }
       reset();
       sent_handshake_count = 0;
+      node_id = fc::sha256();
```

Comment:
Is this caring over a separate fix needed for 1.8? This want part of the original pr.

Reply:
https://github.com/EOSIO/eos/pull/7625/files#diff-3935dcf0fa19e1c59296f2ff874e2621R986